tests/garage/tf/policies/test_categorical_mlp_policy_with_model.py

```diff
@@ -0,0 +1,77 @@
+import pickle
+from unittest import mock
+
+import numpy as np
+import tensorflow as tf
+
+from garage.tf.envs import TfEnv
+from garage.tf.models import Model
+from garage.tf.policies import CategoricalMLPPolicyWithModel
+from tests.fixtures import TfGraphTestCase
+from tests.fixtures.envs.dummy import DummyDiscreteEnv
+
+
+class SimpleMLPModel(Model):
+    """Simple MLPModel for testing."""
+
+    def __init__(self, name, output_dim, *args, **kwargs):
+        super().__init__(name)
+        self.output_dim = output_dim
+
+    def _build(self, obs_input):
+        return tf.constant(
+            1., shape=(
+                1,
+                self.output_dim,
+            ))
+
+
+class TestCategoricalMLPPolicyWithModel(TfGraphTestCase):
+    def setUp(self):
+        super().setUp()
+        self.env = TfEnv(DummyDiscreteEnv())
+        self.env.reset()
+        self.obs, _, _, _ = self.env.step(1)
+        model_path = 'garage.tf.policies.' + \
```

Comment:
split with (), not \

Reply:
when I split with () and put that in `mock.patch()`, it will treat that as a tuple.

Reply:
you don't use commas
```python
(
    'foo'
    'bar'
) == 'foobar'

(
    'foo',
    'bar',
) == ('foo', 'bar')
```